ec/src/models/short_weierstrass/mod.rs

```diff
@@ -77,9 +77,9 @@ pub trait SWCurveConfig: super::CurveConfig {
         item.mul_by_cofactor()
     }
 
-    /// Default implementation of group multiplication for projective
-    /// coordinates
-    fn mul_projective(base: &Projective<Self>, scalar: &[u64]) -> Projective<Self> {
+    /// Standard double-and-add method for multiplication by a scalar.
+    #[inline(always)]
+    fn double_and_add_projective(base: &Projective<Self>, scalar: &[u64]) -> Projective<Self> {
```

Comment:
Ah sorry, I should have been more clear. I meant that this method should be a free generic function outside `SWConfig`. (Ditto for the affine version)